fuzzing/statetest_maker.go

```diff
@@ -51,7 +51,7 @@ type GstMaker struct {
 
 func NewGstMaker() *GstMaker {
 	alloc := make(GenesisAlloc)
-	rnd := common.HexToHash("0x200000")
+	rnd := common.HexToHash("0x000000")
```

Comment:
Need to check so that this doesn't cause problems with existing clients post and pre London. IIRC, there is a good reason why I set the random same as difficulty, and we cannot set difficulty to zero.